arch/xtensa/src/esp32/esp32_spiram.c

```diff
@@ -215,13 +217,14 @@ unsigned int IRAM_ATTR cache_sram_mmu_set(int cpu_no, int pid,
 #ifdef CONFIG_SMP
   spi_enable_cache(1);
 
-  if (smp_start)
+  if (os_ready)
     {
       up_cpu_resume(cpu_to_stop);
     }
 #endif
 
-  leave_critical_section(flags);
+  if (os_ready)
+    leave_critical_section(flags);
```

Comment:
Please include { \n } for all if statements, it is something our CI nxstyle is not catching yet

Reply:
done.

IMO, we should switch to a style which can be enforced with clang-format and drop nxstyle.